minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneClusterControl.java

```diff
@@ -149,9 +141,8 @@ String getJarFromClass(Class<?> clz) {
 
   @Override
   public void adminStopAll() throws IOException {
-    File confDir = getConfDir();
-    String master = getHosts(new File(confDir, "masters")).get(0);
-    String[] cmd = new String[] {SUDO_CMD, "-u", user, ACCUMULO_CONF_DIR + serverAccumuloConfDir, accumuloPath, Admin.class.getName(), "stopAll"};
+    String master = getHosts(MASTER_HOSTS_FILE).get(0);
+    String[] cmd = new String[] {SUDO_CMD, "-u", user, accumuloPath, Admin.class.getName(), "stopAll"};
```

Comment:
> I think we could support it for backwards compatibility in the scripts by replacing the following line:

Oh that would be wonderful. We can look into that separately (since it's not directly related to what you're doing here) if you'd like..